apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -33,7 +37,36 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
     await this.worker.run<CreateHnippNotificationDto>(
       async (message, job): Promise<void> => {
         const messageId = job.id
-        this.logger.info('Message received by worker ... ...', { messageId })
+        this.logger.info('Message received by worker 1.1', { messageId })
```

Comment:
Again, just curious. What does the 1.1 signify here. Is it the version? Is it some formatting magic?

Reply:
just a something i used during dev ... cleaning this up